client/src/components/NoteContainer.jsx

```diff
@@ -189,6 +189,8 @@ export default function NoteContainer() {
         initialContent = "I am a Terminal";
       } else if (editorType === "code") {
         initialContent = "I am a Code Block";
+      } else if (editorType === "image") {
+          initialContent = null;
```

Comment:
If this gets any longer might want to put this in a function and use a switch statement